docs/reference/search/request/from-size.asciidoc

```diff
@@ -1,29 +1,45 @@
-[[request-body-search-from-size]]
-==== From / Size
+[discrete]
+[[paginate-search-results]]
+=== Paginate search results
 
-Pagination of results can be done by using the `from` and `size`
-parameters. The `from` parameter defines the offset from the first
-result you want to fetch. The `size` parameter allows you to configure
-the maximum amount of hits to be returned.
+By default, the search API returns documents for only the top 10 search
+results, starting with the first, most relevant document.
 
-Though `from` and `size` can be set as request parameters, they can also
-be set within the search body. `from` defaults to `0`, and `size`
-defaults to `10`.
+To paginate a larger set of search results, you can use the search API's `size`
+and `from` parameters. The `from` parameter is a zero-indexed offset from
+the first result you want to fetch. The `size` parameter sets the maximum
+number of results to return.
```

Comment:
"Offset from the first result you want to fetch" doesn't make sense. It's an offset from the beginning of the result set. [The Definitive Guide ](https://www.elastic.co/guide/en/elasticsearch/guide/2.x/pagination.html) called it "the number of initial results you want to skip". That seems a little convoluted--it's really just the document you want to start with.

```suggestion
To paginate through a larger set of results, you can use the search API's `size`
and `from` parameters. The `size` parameter is the number of matching documents to return. The `from` parameter is a zero-indexed offset from the beginning of the complete result set that indicates the document you want to start with. 
```